bin/main.rb

```diff
@@ -8,67 +10,73 @@ def print_board(arr)
   puts "#{arr[6]} | #{arr[7]} | #{arr[8]}"
 end
 
-def evaluate_game
-  true
-end
-
-def update_player(player)
-  player
+def checks_input
+  var = gets.chomp
+  until yield(var)
+    puts 'Please enter a valid option'
+    print 'Input value here: '
+    var = gets.chomp
+  end
+  var
```

Comment:
I suggest moving this, too. It could fit on the Game Class and the same idea: return a string and then print it on the `main.rb` file.
The gets.chomp could be changed so that is the parameter of the method.